.github/workflows/publish-artifacts.yml

```diff
@@ -0,0 +1,38 @@
+name: Publish-Artifacts
+
+on:
+  workflow_dispatch:
+  # push:
+  #   branches:
+  #     - main
+  #     - '[0-9]+.[0-9]+'
+  #     - cft-upload-s3
+
+jobs:
+  Cloudformation:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Check if elastic-agent-ec2.yml changed
+        id: changed-elastic-agent-cft
+        uses: tj-actions/changed-files@v35
+        with:
+          files: deploy/cloudformation/elastic-agent-ec2.yml
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-2
+
+      - name: Upload to S3 if elastic-agent CloudFormation template has changed
+        if: steps.changed-elastic-agent-cft.outputs.any_changed == 'true'
+        run: |
+          VERSION=$(grep defaultBeatVersion version/version.go | cut -f2 -d "\"")
+          FILENAME=$(date +"cloudformation-$VERSION-%Y-%m-%d-%H-%M-%S.yml")
+          echo "One or more files in the docs folder has changed."
+          echo "List all the files that have changed: ${{ steps.changed-elastic-agent-cft.outputs.all_changed_files }}"
```

Comment:
I'm not sure why do we need to log this part

Reply:
Will be removed on the next PR while changing the trigger